tests/interfaces/test_batch_autograd.py

```diff
@@ -634,13 +666,14 @@ def cost(x, y, device):
 
         expected = np.array(
             [
-                [[-np.sin(x) / 2, 0], [np.sin(x) / 2, 0]],
+                [[-np.sin(x) / 2, 0], [-np.sin(x) * np.cos(y) / 2, -np.cos(x) * np.sin(y) / 2]],
                 [
-                    [-np.sin(x) * np.cos(y) / 2, -np.cos(x) * np.sin(y) / 2],
+                    [np.sin(x) / 2, 0],
                     [np.cos(y) * np.sin(x) / 2, np.cos(x) * np.sin(y) / 2],
                 ],
             ]
         )

```

Comment:
The expected output changes, since we are now returning the transpose of the Jacobian 🙂 This brings us in line with what `qml.jacobian` expects, and how the Torch interface was already behaving.